storage/objects/generate_v4_get_object_signed_url.go

```diff
@@ -16,38 +16,41 @@ package objects
 
 // [START storage_generate_signed_url_v4]
 import (
+	"context"
 	"fmt"
 	"io"
-	"io/ioutil"
 	"time"
 
 	"cloud.google.com/go/storage"
-	"golang.org/x/oauth2/google"
 )
 
 // generateV4GetObjectSignedURL generates object signed URL with GET method.
-func generateV4GetObjectSignedURL(w io.Writer, bucket, object, serviceAccount string) (string, error) {
+// Signing a URL requires credentials authorized to sign a URL. You can pass
+// these in as a Google service account private key (obtainable from the Google
+// Developers Console) or using a SignBytes function implementing custom signing.
+// In this example, neither of these options are used, which means the following
+// code only works when the credentials are defined via the environment variable
+// GOOGLE_APPLICATION_CREDENTIALS.
```

Comment:
This is not quite accurate, correct? It could also populate signBlob.

I would also maybe move this comment down to where the creds could be passed in via the SignedURLOptions and make it explicit that that's where you would do it.